mjidris/irondb
# Us 114

## Objective
Add navbar component to React application

## Detailed Description
This PR will add a navbar component to the React application, as well as stub out the routes needed.

## Screenshot (if applicable)


## How to Test (if applicable)
In order to test this change, navigate to `irondb/client`, and then run `npm start`.

You will see a navigation bar in the top of the application. In order to change from the authenticated to unauthenticated view (or vice-versa), using your text editor, navigate to `irondb/client/src/index.js`, and on line `16`, change the boolean value for `isAuthenticated` to `true` or `false`.
Full authentication integration will take place in a future US.

You can leave the React server running while you change this value, and the app will auto-reload.

*Note: styles might be a bit messed up, but that can be taken care of in future PRs* 

## Tests (if added)


## Requirements
[US-114](https://tree.taiga.io/project/cbrobsto-iron-db/us/114)
[T-141](https://tree.taiga.io/project/cbrobsto-iron-db/task/141)
[T-149](https://tree.taiga.io/project/cbrobsto-iron-db/task/149)
[T-142](https://tree.taiga.io/project/cbrobsto-iron-db/task/142)
[T-143](https://tree.taiga.io/project/cbrobsto-iron-db/task/143)
